Extensions/xkit_patches.js

```diff
@@ -1294,6 +1515,16 @@ XKit.tools.getParameterByName = function(name){
 
 			},
 
+			/**
+			 * Specify a control button that can be added to future posts
+			 * using `XKit.interface.add_control_button`.
+			 * @param {String} class_name - CSS class of the button to be created
+			 * @param {String} icon - URL of the button's icon
+			 * @param {String} text - Hover text of the button
+			 * @param {Function<Event>} func - Function called on click of control button
```

Comment:
mdn calls this an [EventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventListener)